### DISCUSSION TITLE
Sorry - I don't know how else to say "thank you" for working on this!

### DISCUSSION THREAD

**Discussion by deadrubberboy** [samemory/homebridge-eufy-security-control#17] [General]
Thank you for trying to deliver on Eufy's HomeKit Doorbell empty promise. I had the other plug in working until they started banning my guest account that used it.

Sorry for making a "feature request". Wasn't sure how else to let you know there are many lurkers like me that rely on the kindness of your efforts.

**Comment by jasonwfrancis1** [comment]
I'm so happy for this! I was ready to actually change the camera for one with Homekit support and then this morning saw this had been updated.

**Comment by fullphat** [comment]
Massive thanks from me too - I started following the original work which developed the library that this plugin uses and it's great to now be able to use it.  

I was so disappointed after I realised that the floodlight cam didn't even support RTSP,  let alone HomeKit, and this plugin has let me start the integration of it into my Apple ecosystem - even being able to switch the floodlights on and off using some Aqara buttons around the house has been awesome.

Great job!
